WorkAndFun/src/test/java/pl/infoshare/workandfun/announcements/AnnouncementServiceTest.java

```diff
@@ -162,4 +150,108 @@ void shouldThrowExceptionWhenNotFoundAnnouncementToDeleteById() {
+                        "Andrzej, czyli ja to miłośnik zwierząt chętnie spędzający z nimi czas, nie masz co zrobić ze swoim zwierzakiem, zadzwoń do Andrzeja"),
+                new QuickViewAnnouncementDto(2L, "Mechanik samochodowy",
+                        "do ustalenia indywidualnie", true, LocalDateTime.of(2020, 10, 10, 10, 10), true, "Warszawa, Wola",
+                        "Diagnostyka, wymiana filtrów/oleju, pompowanie kół"),
+                new QuickViewAnnouncementDto(3L, "Przeprowadzki",
+                        "150", false, LocalDateTime.of(2022, 1, 10, 11, 25), false, "Kraków",
+                        "Mam bus, pomogę z przeprowadzką")));
+        doReturn(quickViewAnnouncementDtoList).when(announcementService).findAllSortedByCreateDateDescConvertToDto();
+//        given(announcementService.findAllSortedByCreateDateDescConvertToDto()).willReturn(quickViewAnnouncementDtoList);
+        String parameter = "przeprowadz";
+        //when
+        var actualResult = announcementService.searchAllByParameter(parameter);
+        //then
+        assertThat(actualResult)
+                .hasSize(1)
+                .hasExactlyElementsOfTypes(QuickViewAnnouncementDto.class)
+                .element(0).usingRecursiveComparison().isEqualTo(quickViewAnnouncementDtoList.get(2));
+    }
+
+    @Test
+    void shouldSearchAllByParameterBlank() {
+        //given
+        List<QuickViewAnnouncementDto> quickViewAnnouncementDtoList = new ArrayList<>(List.of(
+                new QuickViewAnnouncementDto(1L, "Wyprowadzam psy, koty, myszy, konie, słonie",
+                        "200", false, LocalDateTime.of(2021, 12, 20, 10, 15), false, "Wrocław, Bielany",
+                        "Andrzej, czyli ja to miłośnik zwierząt chętnie spędzający z nimi czas, nie masz co zrobić ze swoim zwierzakiem, zadzwoń do Andrzeja"),
+                new QuickViewAnnouncementDto(2L, "Mechanik samochodowy",
+                        "do ustalenia indywidualnie", true, LocalDateTime.of(2020, 10, 10, 10, 10), true, "Warszawa, Wola",
+                        "Diagnostyka, wymiana filtrów/oleju, pompowanie kół"),
+                new QuickViewAnnouncementDto(3L, "Przeprowadzki",
+                        "150", false, LocalDateTime.of(2022, 1, 10, 11, 25), false, "Kraków",
+                        "Mam bus, pomogę z przeprowadzką")));
+        doReturn(quickViewAnnouncementDtoList).when(announcementService).findAllSortedByCreateDateDescConvertToDto();
+        String parameter = "";
+        //when
+        var actualResult = announcementService.searchAllByParameter(parameter);
+        //then
+        assertThat(actualResult)
+                .hasSize(quickViewAnnouncementDtoList.size())
+                .hasOnlyElementsOfType(QuickViewAnnouncementDto.class)
+                .usingRecursiveComparison().isEqualTo(quickViewAnnouncementDtoList);
+    }
+
+
+    @Test
+    void shouldFillDB() {
```

Comment:
to jest metoda , która nie wejdzie na produkcje, także nie ma sensu jej trestować

Reply:
racja, usuwam